src/main/java/org/xbill/DNS/Master.java

```diff
@@ -391,6 +399,34 @@ else if ((token.value).charAt(0) == '$') {
 	return rec;
 }
 
+/**
+ * Disable processing of $INCLUDE directives.
+ * When disabled, $INCUDE statements will not be processed.
+ * Depending on the contents of the file that would have been included,
+ * this may cause the zone to be invalid.
+ * (e.g. if there is no SOA or NS at the apex)
+ */
+public void
+disableIncludes() {
+	noExpandIncludes = true;
```

Comment:
Please call the overload instead of directly assigning variables: `disableIncludes(false)`

Reply:
Updated with d9944b2.    What is this project's preferred workflow?   rebase and re-push to my branch?   Or keep both commits?

Reply:
Doesn't really matter to me. I often squash PRs during merge if there's no relevant history.